### PR TITLE
[Term Entry] Java Calendar setFirstDayOfWeek()

### PR DESCRIPTION
**Pull request by alanabarned2**
### Description

- Added the Calendar.setFirstDayOfWeek() term to Java
- Closes Issue #3164 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+The **`.setFirstDayOfWeek()`** method of the `Calendar` class sets what the first day of all weeks will be by accepting an integer value that represents a specific day of the week. 
+
+> **Note:** Most countries consider either Saturday, Sunday, or Monday to be the first day of the week. For example, the United States recognizes Sunday as the first day of the week, while Monday is observed as the first day of the week in India.
+
+## Syntax
+
+```pseudo
+calendar.setFirstDayOfWeek(int day_val)
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md:22]:
```suggestion
calendar.setFirstDayOfWeek(day_val)
```

File: content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  public static void main(String[] args) {
+    // Creates a calendar object.
+    Calendar cal = Calendar.getInstance();
+
+    // Retrieves and prints the current first day of the week. 
+    int firstDay = cal.getFirstDayOfWeek();
+    System.out.println("First day of the week: " + first_day);
+
+    // Changes the first day of the week.
+    int newFirstDay = cal.setFirstDayOfWeek(Calendar.SATURDAY);
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md:44]:
```suggestion
    cal.setFirstDayOfWeek(Calendar.SATURDAY);
    int newFirstDay = cal.getFirstDayOfWeek();
```

File: content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.Calendar;
+
+public class FirstDayOfWeek {
+  public static void main(String[] args) {
+    // Creates a calendar object.
+    Calendar cal = Calendar.getInstance();
+
+    // Retrieves and prints the current first day of the week. 
+    int firstDay = cal.getFirstDayOfWeek();
+    System.out.println("First day of the week: " + first_day);
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/setFirstDayOfWeek/setFirstDayOfWeek.md:41]:
```suggestion
    System.out.println("First day of the week: " + firstDay);
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@alanabarned2 I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@alanabarned2 This is ready for a second review.

**Review by KTom101 — APPROVED** [review]
@alanabarned2 I'm going to keep everything as is. Looks good.

**Pull request merged**